src/hooks/useContentView.js

```diff
@@ -0,0 +1,28 @@
+import { useContext } from 'react';
+import { ContentViewContext } from 'contexts/ContentViewContext';
+
+const useContentView = () => {
+  const {
+    displayedComponent,
+    setDisplayedComponent,
+    displayMap,
+    setDisplayMap,
+    navbarColor,
+    setNavbarColor,
+    navbarLeftButton,
+    setNavbarLeftButton,
+  } = useContext(ContentViewContext);
+
+  return {
+    displayedComponent,
+    setDisplayedComponent,
+    displayMap,
+    setDisplayMap,
+    navbarColor,
+    setNavbarColor,
+    navbarLeftButton,
+    setNavbarLeftButton,
+  };
```

Comment:
you could return the result of `useContext(ContentViewContext)` call. I'm worried that since you are returning a plain object on each call it might vary (not a 100% sure but there might be that risk)
```suggestion
  return useContext(ContentViewContext);
```

Reply:
Done